src/jobservice/job/impl/gc/garbage_collection.go

```diff
@@ -192,17 +191,19 @@ func (gc *GarbageCollector) init(ctx job.Context, params job.Parameters) error {
 // cleanCache is to clean the registry cache for GC.
 // To do this is because the issue https://github.com/docker/distribution/issues/2094
 func (gc *GarbageCollector) cleanCache() error {
-	con, err := redis.DialURL(
-		gc.redisURL,
-		redis.DialConnectTimeout(dialConnectionTimeout),
-		redis.DialReadTimeout(dialReadTimeout),
-		redis.DialWriteTimeout(dialWriteTimeout),
-	)
-
+	pool, err := cache.GetRedisPool("GarbageCollector", gc.redisURL, &cache.RedisPoolParam{
+		PoolMaxIdle:           0,
+		PoolMaxActive:         1,
+		PoolIdleTimeout:       60 * time.Second,
+		DialConnectionTimeout: dialConnectionTimeout,
+		DialReadTimeout:       dialReadTimeout,
+		DialWriteTimeout:      dialWriteTimeout,
```

Comment:
Can we unify the Redis param in harbor core?

Reply:
This PR is focus on support redis sentinel, so I don't want to optimize the param too much for each components, as this might introduce new issues.